samza/notification-job/src/main/config/local.notification-job.properties

```diff
@@ -1,6 +1,6 @@
 # Job
 job.factory.class=org.apache.samza.job.yarn.YarnJobFactory
-job.name=local.notification.all
+job.name=local.lms.notification
```

Comment:
is this needed for all or just for your dev env.

Reply:
This is needed for local. For dev it is separate config file.

Reply:
this file affects only local environment.